domain/schema/model/sql/0033-operation.sql

```diff
@@ -127,3 +127,14 @@ CREATE TABLE operation_task_log (
     FOREIGN KEY (task_uuid)
     REFERENCES operation_task (uuid)
 );
+
+-- operation_parameter holds the parameters passed to an operation.
+CREATE TABLE operation_parameter (
```

Comment:
issue: this table is ambiguous and duplicates the contents of the `operation_exec` table.